Gandalan.IDAS.WebApi.Client/BusinessRoutinen/MailWebRoutinen.cs

```diff
@@ -24,12 +24,9 @@ public async Task<JobStatusResponseDTO> Send(MailJobInfo job, List<string> attac
                 foreach (var attachment in attachments)
                 {
                     // read each file and add it to the multipart form data
-                    var fileContent = new ByteArrayContent(File.ReadAllBytes(attachment));
-                    fileContent.Headers.ContentDisposition = new System.Net.Http.Headers.ContentDispositionHeaderValue("attachment")
-                    {
-                        FileName = Path.GetFileName(attachment)
-                    };
-                    content.Add(fileContent, "files", Path.GetFileName(attachment));
+                    var fileStream = File.OpenRead(attachment);
```

Comment:
Please use `using` to close stream

Reply:
Agreed, other than that it's fine

Reply:
Cannot do any "using" in this parts of the code as I am getting "System.ObjectDisposedException: Cannot access a disposed object." on higher level.